ql/models/shortrate/onefactormodels/gsr.cpp

```diff
@@ -92,6 +92,12 @@ Gsr::Gsr(const Handle<YieldTermStructure> &termStructure,
     initialize(T);
 }
 
+void Gsr::update() { 
```

Comment:
@pcaspers , if you don't have time, I would not worry about it. The changes to gsr are not really going to make a noticeable difference. It is just a habit that when I am reading code, and am trying to understand every step, not just trying to get a quick overview, then I tend to play around and make that sort of small refactoring moves.